docs/hub/post_install.md

```diff
@@ -35,4 +35,4 @@ The following process must run on a host that supports installation of the **hzn
 
 ## What's Next
 
-After completing the steps in [Post installation configuration](#postconfig), follow the process on the [Gather edge node files](gather_files.md) page to prepare installation media for your edge nodes.
+Follow the process on the [Gather edge node files](gather_files.md) page to prepare installation media for your edge nodes.
```

Comment:
I think we may actually have a different technique now that is much easier.

Reply:
OK - Is the new installation technique documented somewhere?  Do we need a separate GH issue for that?
This merge just cleans up the grammar to match the IEAM documentation.

Reply:
I believe this is the docs for adding more edge agents to an existing installation: https://github.com/open-horizon/devops/tree/master/mgmt-hub#adding-more-edge-devices

Reply:
Wow - The README over in the devops repo really needs to move over to OH documentation.
https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md

That is a great page that is not described anywhere in the OH documentation. Let's fork that work effort into a separate Issue.  See #224